Documentation/ceph-upgrade.md

```diff
@@ -256,7 +256,7 @@ below, which you should change to match where your images are located.
 ```yaml
     env:
     - name: ROOK_CSI_CEPH_IMAGE
-        value: "quay.io/cephcsi/cephcsi:v1.2.1"
+        value: "quay.io/cephcsi/cephcsi:v1.2.2"
```

Comment:
Should the version change to v1.2.2 be in a separate commit or PR from the new settings for CSI driver registration? For example, are there any critical fixes in v1.2.2 so we should backport that?